packages/react/src/components/Storage/StorageManager/utils/uploadFile.ts

```diff
@@ -29,12 +29,18 @@ export function uploadFile({
   if (isResumable === true) {
     return Storage.put(fileName, file, {
       level,
-      resumable: isResumable,
+      resumable: true,
```

Comment:
If this is intentional (which i think it is), can we please add a comment explaining why we default to `true`? Could be confusing to come across this in the future